pool/pool.py

```diff
@@ -83,7 +84,10 @@ def __init__(
         self.config = config
         self.constants = constants
 
-        self.store: AbstractPoolStore = pool_store or SqlitePoolStore()
+        if "store" in pool_config == "MariadbPoolStore":
```

Comment:
I think it worked by luck.

you probably meant:
`if pool_config.get('store') == "MariadbPoolStore":`

Reply:
lol yep thats what i meant

Reply:
fixed it + fixed the logging